### ISSUE TITLE
Feature Request: Select/Unselect All

### ISSUE THREAD

**Issue by devsideal** [broakenmedia/MultiContactPicker#11]
Nice work by the library, but It would be very help full if add a option (may be optional) to Select/Un select All contacts.

**Comment by broakenmedia** [comment]
Good idea, any suggestions how it might be implemented in UI though? Space is pretty limited.

**Comment by devsideal** [comment]
I have implemented this option and made some changes in UI,
Please check my PR and merge if you like those changes.

**Comment by broakenmedia** [comment]
Thanks for your pull request, i have gone ahead and implemented the feature now however in a slightly different way. It will be available on the next release!
